sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -61,29 +61,25 @@ class ArtifactManifestProperties(object):  # pylint: disable=too-many-instance-a
     :ivar bool can_read: Read Permissions for an artifact.
     :ivar bool can_list: List Permissions for an artifact.
     :ivar architecture: CPU Architecture of an artifact.
-    :vartype architecture: Optional[~azure.containerregistry.ArtifactArchitecture]
+    :vartype architecture: Optional[str]
     :ivar created_on: Time and date an artifact was created.
     :vartype created_on: Optional[~datetime.datetime]
     :ivar Optional[str] digest: Digest for the artifact.
     :ivar last_updated_on: Time and date an artifact was last updated.
     :vartype last_updated_on: Optional[~datetime.datetime]
     :ivar operating_system: Operating system for the artifact.
-    :vartype operating_system: Optional[~azure.containerregistry.ArtifactOperatingSystem]
+    :vartype operating_system: Optional[str]
```

Comment:
This feels like it could be breaking -- can you double-check? 

For example, I used to be able to do `operating_system.value` on the enum, but can't with plain string

Reply:
Yeah this is a problem....
Looking at the linked issues, in both cases the service is actually returning the literal value "unknown" - in which case, why isn't this being added to the enum?
The other possibility, if we are could potentially get other random, unrecognized values here, is to expose this as string, without changing the behaviour for existing values, e.g.
```python
if self._architecture is not None:
    try:
        self._architecture = ArtifactArchitecture(self._architecture)
    except ValueError:
        pass
```
This would still need a type hint/docstring update, so is still a little awkward, especially for the scenario @kristapratico mentioned.

Reply:
@kristapratico Yes, it will be a breaking change. I didn't think about it.

After re-thinking the breaking change scenario, I'll prefer to ignore the `ValueError` which the other possibility @annatisch mentioned. 
The reason is, - for customers not using "unknown" will have no effect, - for customers using "unknown" is blocking now, we can let them know that they should do some updates in their end when reading properties values as the new value is in type string after updating the sdk version. The breaking change will not happen silently.

Reply:
Update: got response from service team that they have no concerns for adding "unknown" to the known value list. It looks we can fix the issue without breaking change.

Reply:
Update: after talking with v-team, it's not worth add "unknown" to swagger - because that will bring more cost on code regeneration across languages, and need to deal with every new values the same in future. It's also not worth to add "unknown" to the enums in customer layer.